SmartSyncExplorerReactNative/installios.js

```diff
@@ -6,7 +6,7 @@ var path = require('path');
 var fs = require('fs');
 
 console.log('Installing sdk dependencies');
-var sdkDependency = 'salesforcemobilesdk-ios';
+var sdkDependency = 'SalesforceMobileSDK-iOS';
```

Comment:
This was the only template using lowercase and it was causing path issues while copying resources. I lined it up to be consistent with the other templates.